src/apps/components/run-header.js

```diff
@@ -94,6 +119,8 @@ class RunHeader extends React.Component {
         };
 
         let juttleViewer = this.props.bundle && this.state.showJuttle ? <JuttleViewer bundle={this.props.bundle} /> : false;
+        
+        let debugViewer = this.state.showDebug ? <LogExplorer logLines={this.props.logLines} /> : false;
```

Comment:
Whereas this is typically how we toggle the visibility of a component in React, the unfortunate side-effect is it causes the log explorer to lose its state when its mounted/unmounted. This means that if somebody opens the log explorer, searches, moves the cursor, closes it, and reopens it, they will lose their position in the log.

Two possible workarounds
- s the state in a redux.
 - downside is that it makes the LogExplorer component itself less reusable
- use `display: none;`
 - the downside is that a bit more DOM sticks around even when its closed

@mnibecker and I vote for `display: none;`.